docs/reference/analysis.asciidoc

```diff
@@ -25,7 +25,7 @@ into these terms, which would be added to the inverted index.
 
 [source,text]
 ------
-[ quick, brown, fox, jump, over, lazi, dog ]
+[ quick, brown, fox, jump, over, lazy, dog ]
```

Comment:
This is actually correct. The english analyzer includes a stemming token filter which will index the token `lazi` instead of `lazy` since `lazi` is the root form. see https://www.elastic.co/guide/en/elasticsearch/reference/7.x/analysis-lang-analyzer.html#english-analyzer for more information on what is included in the english analyzer

Reply:
I see, didn't know that lazi was the root form. Thanks for the info!